docker-compose.yml

```diff
@@ -2,29 +2,23 @@ version: "3"
 
 services:
   mindwendel:
-    container_name: mindwendel
-    build:
-      context: .
-      target: development
+    image: ghcr.io/mindwendel/mindwendel:latest
     environment:
+      # You can also define the environment variables in the .env.prod file. Use the `.env.prod.default` as template
+      # When executing `docker up`, then `.env` file will be considered automatically
+      # env_file:
+      #  - .env.prod
       DATABASE_USER: postgres
       DATABASE_USER_PASSWORD: postgres
       DATABASE_NAME: mindwendel
       DATABASE_PORT: 5432
       DATABASE_HOST: db
+      URL_HOST: "localhost"
+      SECRET_KEY_BASE: "zva4uKXd0O/gN/LybopmApQkT7iZnnyCK8JUh95U3Go65k9umxwxXwH/7gPSOdHi"
```

Comment:
why do we have the secret key base at two places? (env.prod and docker-compose.yml which is prod)

Reply:
ah ok you have to decide for the env file or stating the env variables here...

Reply:
I know you want to help people here. But I would remove it from the prod compose file. Why? 90% of all the people using this will not change it. So this will result in a security flaw in many instances. 

Ideas:
- Add some description in the readme, how to generate one (without elixir)
- Maybe generate and print it to console if none is set
- ...

Reply:
I described it in the installation guide.